accounts/queries.py

```diff
@@ -118,14 +118,24 @@ def get_child_participation_eligibility(child, study) -> bool:
     must_not_have_participated = True
 
     if study.must_have_participated.exists():
-        must_have_participated = child.responses.filter(
-            study__in=study.must_have_participated.all()
-        ).exists()
+        must_have_participated_all = [
+            i
+            for i in study.must_have_participated.all()
+            if child.responses.filter(study__exact=i).exists()
+        ]
```

Comment:
I am not going to hold up the PR for this, but `__exact` seems needless.  Are you finding the query is returning different result with and without it?

Reply:
You're right, `__exact` isn't needed. I think the code that you're commenting on here might be outdated, because I think I fixed all of these. I'll double check and remove any that I missed.